pkg/chartutil/capabilities.go

```diff
@@ -28,12 +28,10 @@ import (
 	helmversion "helm.sh/helm/v3/internal/version"
 )
 
-const (
+var (
 	k8sVersionMajor = 1
 	k8sVersionMinor = 20
-)
 
-var (
```

Comment:
Since this starts with a capital letter, this makes it have global visibility. Someone using the SDK could modify this variable, whereas before this wasn’t possible due to it being declared a `const`. I don’t think mutability is a desirable design choice.

Reply:
Meh… prior to this change it was declared a `var` so this doesn’t change its existing functionality. I don’t agree with this design choice, but I guess it is what it is /shrug